fragment.js

```diff
@@ -79,7 +79,7 @@
     // If the innerHTML is nonempty: the context is interpreted as the
     // combination of the JSONified innerHTML and the existing context.
     // The JSONified innerHTML has a higher precedence over the existing context.
-    if (element.innerHTML != "") {
```

Comment:
+1, I ran into this with the following:

`<section id="abstract" data-fragment="abstract.html"> </section>`

Trying to debug why the `element.innerHTML=""` was doing the wrong thing was painful.